DependencyInjection/FOSElasticaExtension.php

```diff
@@ -2,13 +2,13 @@
 
 namespace FOS\ElasticaBundle\DependencyInjection;
 
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
```

Comment:
oops, it seems phpStorm had "Optimize imports" option enabled, reverted this part.